src/settings.js

```diff
@@ -413,11 +413,21 @@ var GL_EMULATE_GLES_VERSION_STRING_FORMAT = 1;
 // format, but also in desktop/mobile GLES/GL extension format with "GL_" prefix.
 var GL_EXTENSIONS_IN_PREFIXED_FORMAT = 1;
 
-// If true, adds support for automatically enabling all GL extensions for
-// GLES/GL emulation purposes. This takes up code size. If you set this to 0,
-// you will need to manually enable the extensions you need.
+// If true, adds support for automatically enabling GL extensions for GLES/GL
+// emulation purposes. This takes up code size. If you set this to 0,  you will
+// need to manually enable the extensions you need.
 var GL_SUPPORT_AUTOMATIC_ENABLE_EXTENSIONS = 1;
```

Comment:
With the comment below, it seems like there would be three distinct modes here. Could they be all merged into this option?

- `0`: don't automatically enable any extensions
- `1`/`"native"`: enable extensions for GLES/GL emulation (is that exactly what it's for?)
- `"all"`: all available extensions